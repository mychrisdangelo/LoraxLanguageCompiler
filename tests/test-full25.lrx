/* 
 * Authors:
 * Chris D'Angelo
 * Kira Whitehouse
 * Tests multiple child operator on lhs and rhs.
 */

int main()
{
	tree <int> t(2);
	tree <int> s(2);

	t = 1[2[3, 4], 5];
	s = 6[7[8,9], 10[]];

	t%0%0 = s;
	print(t, '\n');
	t = s%0%0;
	print(t, '\n');
}

