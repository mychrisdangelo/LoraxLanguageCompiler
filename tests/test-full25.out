1[2[6[7[8[null,null],9[null,null]],10[null,null]],4[null,null]],5[null,null]]
8[null,null]
